packages/Button/src/LinkButton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+// import iconButtonStyles from "./IconButton.styles";
+import Button from "./Button";
+
+const LinkPropTypes = {
+  children: PropTypes.node.isRequired,
+  kind: PropTypes.oneOf(Button.Kinds.ALL),
+  size: PropTypes.oneOf(ShirtSizes.MEDIUM),
+  url: PropTypes.string,
+};
+
+const LinkDefaultProps = {
+  kind: Button.Kinds.DEFAULT,
+  size: ShirtSizes.MEDIUM,
+  url: null,
+};
+function redirect_to(url) {
+  window.location = url;
+}
+const LinkButton = React.forwardRef((props, ref) => {
+  const buttonProps = {
+    children: null,
+    icon: props.children,
```

Comment:
Try not to copy paste too much. For learning purposes its good to understand what you are copy pasting. For example this line wouldn't make sense here.